src/tests/banners/bannerSelection.test.ts

```diff
@@ -0,0 +1,39 @@
+import { readerRevenueRegionFromCountryCode, redeployedSinceLastClosed } from './bannerSelection';
+
+describe('readerRevenueRegionFromCountryCode', () => {
+    it('should return a region', () => {
+        const unitedKingdom = readerRevenueRegionFromCountryCode('GB');
+        expect(unitedKingdom).toBe('united-kingdom');
+
+        const germany = readerRevenueRegionFromCountryCode('DE');
+        expect(germany).toBe('european-union');
+
+        const australia = readerRevenueRegionFromCountryCode('AU');
+        expect(australia).toBe('australia');
+
+        const fiji = readerRevenueRegionFromCountryCode('FJ');
+        expect(fiji).toBe('rest-of-world');
+
+        const usa = readerRevenueRegionFromCountryCode('US');
+        expect(usa).toBe('united-states');
+    });
+});
+
+describe('redeployedSinceLastClosed', () => {
```

Comment:
It looks as though we're not mocking the dependencies of `bannerSelection.tsx` in this test, so are we using the actual implementation? We don't want to make a fetch request to the redeploy end point in the test, we should just mock the `cacheAsync` dependency, ideally with two return values one where  `redeployedSinceLastClosed` would return `true` and the other `false`, based on the `engagementBannerLastClosedAt` in the `targeting` object here.